packages/backend/src/routes/dons.ts

```diff
@@ -0,0 +1,77 @@
+import express, { Request } from 'express';
+import prisma from '@/lib/prismaClient.js';
+
+interface Topping {
+  id: number,
+  amount: number,
+}
+
+interface PriceBody {
+  size: number,
+  toppings: Topping[] | undefined,
+}
+
+const router = express.Router();
+
+router.post('/price', async (req: Request<any, any, PriceBody>, res, next) => {
+  const size = req.body.size;
+  const toppings = req.body.toppings || [];
+
+  if (!size) {
+    next();
+    return;
+  }
+
+  try {
+    const sizes = await prisma.sizes.findFirst({ 
```

Comment:
```suggestion
    const size = await prisma.sizes.findUnique({ 
```
[nits] 
- 戻ってきたオブジェクトが配列 ([]size) ではなくて, size であることがわかる変数名にしたい
- findUnique と findFirst の違いは統一性の観点から一応 (戻り値の型は同じだし, index 貼ってるので速度もほぼ変わらないので nits)

Reply:
[91ccad4](https://github.com/ARCircle/Kojirer/pull/33/commits/91ccad449e671b010fe7d44cc6fffbfa006f11b7)で直しました

Reply:
`req.query`のところを`_size`, `_topping`とするのはどうでしょう